tools/webhook-endpoint/README.md

```diff
@@ -0,0 +1,17 @@
+# Webhook App Engine Tool
+## Webhook App Overview
+The webhoook tool is intended to be an easy to deploy and easy to manage
+application.  You can use it to recieve and process any un-authenticated http(s)
+data.
```

Comment:
At first glance, un-auth'd data ingest seems like a great way to have a bad actor drive your cloud bill through the roof.
Can we please describe a motivating use case?

Reply:
This was a feature of alooma which we are trying to supply an open-source solution for.  Generally the most common use case here is a Mobile or Javascript SDK.  These are all open un-authenticated endpoints, so you could in theory go into someone's webpage or mobile app and abuse the web traffic.
This can also be used to replicate SaaS webhook data in realtime (ie. Stripe supplies webhooks like triggers which you can use to receive data in realtime)

This was always a nice way to allow different team to build their own integrations as well.  Often people wanted to build the reader and then just shoot data at Alooma's endpoint to manage creating and managing tables.  This option would be doable here w/ private IPs only for such use cases.

I think overall you'd have to lean on the DOS that's built into App Engine in this event.  The plan is to also use tokens which translate (base64) to the destination as a light extra layer.  Unknown or requests without tokens would be ignored.